via/static/scripts/test-util/wait.ts

```diff
@@ -30,6 +30,41 @@ export async function waitFor<T>(
   });
 }
 
+// Minimal Enzyme types needed by `waitForElement`.
```

Comment:
Enzyme doesn't have its own types, and the third-party types pull in all of React's types. These minimal definitions are good enough for now. I'm happy to write Preact-oriented types for the parts of Enzyme we use when we need them.